packages/babel-plugin-transform-es3-reserved-words/src/reserved-words.js

```diff
@@ -0,0 +1,96 @@
+const es2017 = [
```

Comment:
It would be great to add a test that checks each reserved word. We _may_ be able to use the same helper we do in the other `es3` transforms, but you'll have to verify. https://github.com/babel/babel/blob/5f285c1034acebf9303018a541831da143823cc4/packages/babel-plugin-transform-es3-property-literals/src/index.js#L10

Reply:
As far as I can tell the helper from `babel-types` uses `esutils.keyword` and only checks for ES5 and ES6, but not ES3 keywords. That would mean that other ES3 plugins actually don't respect all ES3 keywords?

Reply:
Hmm, interesting. I think I'd be in favor of adding another utility to `babel-types` alongside the current one, but for ES3 identifiers. Then we can use that function across all three transforms and delete this file.